huntsman/tests/test_huntsman.py

```diff
@@ -252,8 +252,6 @@ def test_entering_darks_state(pocs, db):
 
     pocs.state = 'taking_darks'
 
-    # assert(pocs.state == 'parked')
-
 
 def test_darks_collection_simulator(pocs, tmpdir):
```

Comment:
- It would be nice to include a short description of what this test is trying to achieve.
- What is the purpose of `return False` at the end of this function?